zcash_primitives/src/transaction/builder.rs

```diff
@@ -499,7 +504,7 @@ impl<R: RngCore + CryptoRng> Builder<R> {
                 return Err(Error::NoChangeAddress);
             };
 
-            self.add_sapling_output(change_address.0, change_address.1, change, None)?;
+            self.add_sapling_output::<Network>(change_address.0, change_address.1, change, None)?;
```

Comment:
`add_sapling_output` no longer has to specify the type parameter, since it is already specified in `impl<P: consensus::Parameters, R: RngCore + CryptoRng> Builder<P, R>`.